woocommerce/woocommerce
# add HPOSToggleTrait.php to unit test loader

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

This PR fixes two unit test files that could not be run individually by adding `HPOSToggleTrait.php` to the unit test boot loader. It also does a bit of namespace cleanup in HPOSToggleTrait.php. 

### How to test the changes in this Pull Request:

Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Install unit tests locally `tests/bin/install.sh`
2. Run `for T in $(find ./tests/php -name '*Tests.php'); do vendor/bin/phpunit $T; done` in `trunk`
3. You will see errors like
```
Fatal error: Trait “Automattic\WooCommerce\RestApi\UnitTests\HPOSToggleTrait” not found in /Users/ronrennick/Sites/wc/wp-content/woocommerce/plugins/woocommerce/tests/php/src/Internal/DataStores/Orders/OrdersTableRefundDataStoreTests.php on line 14
```
5. Switch to this branch
6. Repeat step 2
7. No errors will occur
8. Verify all unit tests run in CI or run locally with `vendor/bin/phpunit`
